cmd/dex/serve.go

```diff
@@ -277,6 +277,18 @@ func serve(cmd *cobra.Command, args []string) error {
 		logger.Infof("config device requests valid for: %v", deviceRequests)
 		serverConfig.DeviceRequestsValidFor = deviceRequests
 	}
+	refreshTokenPolicy, err := server.NewRefreshTokenPolicyFromConfig(
```

Comment:
Thought about something similar. Unfortunately, this is not possible with the current design. We have to do cross imports, which is not allowed. I will remove the `FromConfig` suffix because it's not true.